plugins/hydra_ax_sweeper/tests/test_ax_sweeper_plugin.py

```diff
@@ -134,7 +127,7 @@ def test_jobs_configured_via_config(sweep_runner: TSweepRunner) -> None:
         assert returns["optimizer"] == "ax"
         assert len(returns) == 2
         best_parameters = returns["ax"]
-        assert len(best_parameters) == 2
+        # assert len(best_parameters) == 2
```

Comment:
nope.

Reply:
The len tests are removed because any params passed via cmd also go to ax-plugin and appear in `best_parameters`

Reply:
I don't follow why it's removed but I am just responding to the commented out code.
if you need it, make it code. if you don't need it, remove it.